localization-pack-tr/StringTableTrTreatments.xml

```diff
@@ -1,287 +1,284 @@
-﻿<Database>
-    <GameDBStringTable ID="LOC_EN_TREATMENTS">
+<Database>
+    <GameDBStringTable ID="LOC_TR_TREATMENTS">
 
-        <LanguageCode>en</LanguageCode>
-        <LanguageNameLocalized>English</LanguageNameLocalized>
+        <LanguageCode>tr</LanguageCode>
+        <LanguageNameLocalized>Turkish</LanguageNameLocalized>
 
         <Contributors>
             <Name>Oxymoron Games</Name>
```

Comment:
You can also remove Oxymoron, you are the translator of these files.

Reply:
:) ok